test/delivery_test.rb

```diff
@@ -22,6 +22,16 @@
       -> { MissingToMailer.deliver }.must_raise Lotus::Mailer::MissingDeliveryDataError
     end
 
+    it "Lotus::Mailer::MissingDeliveryDataError bubles up the original Mail::ArgumentError message" do
```

Comment:
What do you think about rewording it so that a verb follows after `it`.

s/bubles/bubbles